apps/dispatch-service/src/file/file.service.ts

```diff
@@ -82,47 +83,32 @@ export class FileService {
                 file.location.path != '.',
             )
             .map((file: CombineArchiveManifestContent) => {
-              const fileS3Path = this.filePaths.getSimulationRunContentFilePath(
-                id,
-                file.location.path,
-              );
-              const fileUrl =
-                this.filePaths.getSimulationRunFileContentEndpoint(
-                  false,
-                  id,
-                  file.location.path,
-                );
-              const fileProperties: Promise<AWS.S3.HeadObjectOutput> =
-                this.storage
-                  .getFileProperties(fileS3Path)
-                  .catch((error: any) => {
-                    throw new InternalServerErrorException(
-                      `The size of '${
-                        file.location.path
-                      }' for simulation run '${id}' could not be retrieved: ${this.getErrorMessage(
-                        error,
-                      )}`,
+              const fileInfo = this.storage
+                .getFileInfo(id, file.location.path)
+                .pipe(
+                  map((fileInfo: FileInfo): ProjectFileInput => {
+                    // TODO do we want to throw here if we don't have a size?
```

Comment:
@jonrkarr You were throwing an error here if we were unable to get the file size. Is this something that we should do here? I am currently just setting the size to undefined if we are not able to get it, as it feels silly to fail the whole run just because one of the file sizes is missing. 
What do you think?